src/layouts/JoinForm.astro

```diff
@@ -0,0 +1,145 @@
+---
+import Countries from "@data/Countries.mjs";
+---
+
+<script blocking="render" async is:inline>
+	const getElement = document.getElementById.bind(document);
+	const EMAIL_REGEX = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+	let token;
+	function setToken(newToken) {
+		token = newToken;
+	}
+
+	function submit() {
+		getElement("btn-submit").classList.add("is-loading");
+		getElement("error-token").classList.add("is-hidden");
+		getElement("error-submit").classList.add("is-hidden");
```

Comment:
These already are hidden in the HTML. Is JS just that broken?

Reply:
The `is-hidden` class might have been removed through a previous `submit()` call.